File: VB/SampleCore/Models/XPEmailModel.vb
Imports Newtonsoft.Json
Imports System
Imports System.ComponentModel.DataAnnotations

Namespace SampleCore.Models
    Public Class XPEmailModel

        Public Property ID() As Integer
        Public Property Subject() As String
        Public Property From() As String
        Public Property Sent() As Date
        Public Property Size() As Long
        Public Property HasAttachment() As Boolean
    End Class
End Namespace

File: CS/SampleCore/Models/XPEmailModel.cs
using Newtonsoft.Json;
using System;
using System.ComponentModel.DataAnnotations;

namespace SampleCore.Models {
    public class XPEmailModel {

        public int ID {
            get;
            set;
        }
        public string Subject {
            get;
            set;
        }
        public string From {
            get;
            set;
        }
        public DateTime Sent {
            get;
            set;
        }
        public long Size {
            get;
            set;
        }
        public bool HasAttachment {
            get;
            set;
        }
    }
}